package.json

```diff
@@ -2,9 +2,18 @@
   "name": "aspxauth",
   "version": "1.0.0",
   "description": "Verify and decrypt .NET's .ASPXAUTH cookie from node",
-  "main": "index.js",
+  "main": "lib/index.js",
+  "files": [ "lib" ],
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "babel src -d lib",
+    "format": "jscs --fix ./",
+    "lint": "eslint --fix ./",
+    "pretest": "npm run format && npm run lint",
+    "test": "mocha --require spec/babelhook spec/*.spec.js",
+    "test:watch": "npm test -- -w",
```

Comment:
Add `test:only` that looks like the current `test` and update `test` to just call `cover`?